modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -323,6 +323,17 @@ function serializeBody(body: BodyInit | null): StructuredClonable {
 	} else {
 		// warn about unserializable body type (eg. FormData, search params,
 		// readable stream),
```

Comment:
I think we need to update this comment, or maybe just get rid of it... it is a "TODO" in disguise.

So:

```js
if (
    body === null
    // etc
) {
    return body;
}
else if (body instanceof FormData) {
    // your implementation
}
else {
    console.warn(
       `Unsupported body type: ${typeof body}`
    );
}
```

`typeof body` is probably not the greatest descripition, but it is easy at least.

Reply:
yes that looks better, thanks.